docs/lsm/partial_compiles/partial_compiles.rst

```diff
@@ -6,40 +7,63 @@ Partial compilation is an approach to speed up compilation when the Service Inve
 
 Ordinarily, LSM re-compiles all instances on every update. This means that as the inventory grows, the compiles become slower. Partial compiles allow LSM to re-compile only those instances that are relevant to the current service instance, avoiding any slowdown. 
 
+Implementation guidelines
+-------------------------
+
+1. for every :inmanta:entity:`lsm::ServiceEntity`, 
+
+    1. make sure to collect all resources it contains in the relation :inmanta:relation:`owned_resources<lsm::ServiceBase.owned_resources>`
+    2. make sure to always enable the select the ``parent`` implementations (`implement ... using parents`)
+2. for every :ref:`Inter Service Relation<inter_service_relations>`
+    
+    1. indicate which side of the relation is the owner by setting :inmanta:attribute:`lsm::ServiceEntityBinding.relation_to_owner` and :inmanta:relation:`lsm::ServiceEntityBinding.owner`. 
```

Comment:
This implies there is a choice but there isn't, is there? If I recall correctly inter-service relations are unidirectional on the API level, so there would be only one direction possible?

Reply:
I tried to reformulate